### ISSUE TITLE
Timed-Metadata for Guided Transcoding

### ISSUE THREAD

**Issue by XavierDX** [Dash-Industry-Forum/Ingest#88]
Dear experts,
I would like to inform you that at the 127th MPEG meeting, a proposal of content-dependent Metadata for Content-Guided Transcoding was provided in output document N18637 "Technologies under Consideration for ISO/IEC 23001-10 - Timed-Metadata Metrics of Media in ISO BMFF". The use case is described in DIS of ISO/IEC 23090-8 Network Based Media Processing (NBMP) standard or in the future ISO/IEC 23090-11 Implementation Guidelines for Network-based Media Processing. I think that these Metadata can be valuable for the Ingest specification your are currently developing. I thank you for considering them in your work and would appreciate some feedback from you. Thanks a lot 

I think that these Metadata 
--

**Comment by RufaelDev** [comment]
tf oct 17 2019 we need more input

**Comment by XavierDX** [comment]
Dear experts,
What kind of inputs do you need a part from the Use case description and the metrics themselves?
Thanks

**Comment by RufaelDev** [comment]
Dear Xavier, DASH-IF doesn't have formal access to the documents you refer to and therefore it can't understand your proposal, one first step would be to clearly describe the proposal (the use case is more or less clear), and a second to see how it fits this specification. For the latter it would be best to join one of the conference calls, thanks

**Comment by ZmGorynych** [comment]
@XavierDX I am strongly against this. 

We will be adding an extra use case where you have a transcoder origin, which is possible but not mainstream. DASH already allows 23001-10 carriage in a separate AdaptationSet and we do not disallow this anywhere in the spec. We can mention  that there may be various types of metadata (KLV, quality metadata, transcoding metadata, upcoming ARI tracks in 6th edition, etc), but keep it informational and not specify or require any processing model for them.

**Comment by ZmGorynych** [comment]
TLDR; we want to say "encoder can produce and post additional metadata adaptation sets". We can further require the receiver to respond with a 5xx to any attempt to post unsupported metadata so that the encoder would know that its output goes to /dev/null.

**Comment by XavierDX** [comment]
@ZmGorynych I was invited last year to inform the DASH-IF group in charge of Ingest specification about the MPEG initiative of bringing Metadata to guide a Transcoding process. If the Transcoding Use case does not fit in your current specification work, you can ignore it. In any case, these Metadata are purely informational and I am happy if they are considered at the same level as quality Metadata.

**Comment by RufaelDev** [comment]
it has been a long time no contributions we will close it for now
